app/main/hotkeys.js

```diff
@@ -33,6 +33,7 @@ const HOTKEYS = new Map([
   ['CmdOrCtrl+6', bw => () => bw.webContents.send('select-tab-index', 5)],
   ['CmdOrCtrl+7', bw => () => bw.webContents.send('select-tab-index', 6)],
   ['CmdOrCtrl+8', bw => () => bw.webContents.send('select-tab-index', 7)],
+  ['CmdOrCtrl+9', bw => () => bw.webContents.send('select-tab-index', -1)], // last tab
```

Comment:
Why not send a `select-last-tab` action instead of complicating the `select-tab-index` actions?